tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Transports/LargePayload/DefaultTransportLargePayloadTests.cs

```diff
@@ -0,0 +1,28 @@
+// <copyright file="DefaultTransportLargePayloadTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_1
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    [Collection(nameof(DefaultTransportLargePayloadTests))]
+    public class DefaultTransportLargePayloadTests : LargePayloadTestBase
```

Comment:
If you want to stop parallilisation, you'll need this too:
```suggestion
    [Collection(nameof(DefaultTransportLargePayloadTests))]
    [CollectionDefinition(nameof(DefaultTransportLargePayloadTests), DisableParallelization = true)]
    public class DefaultTransportLargePayloadTests : LargePayloadTestBase
```